camlidl.opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "camlidl"
+version: "1.11"
+synopsis: "Stub code generator for OCaml"
+description: """\
+CamlIDL is a stub code generator for OCaml. It automates the
+generation of C stub code required for the Caml/C interface, based on
+an MIDL specification. Also provides some support for Microsoft's COM
+software components."""
+maintainer: "Nicolas Berthier <m@nberth.space>"
+authors: "Xavier Leroy"
+license: [
+  "QPL-1.0 WITH OCaml-LGPL-linking-exception"
+  "LGPL-2.0-or-later WITH OCaml-LGPL-linking-exception"
+]
+homepage: "https://github.com/xavierleroy/camlidl"
+bug-reports: "https://github.com/xavierleroy/camlidl/issues"
+depends: [
+  "ocaml" {>= "4.05"}
+]
+build: [
+  ["mv" "config/Makefile.unix" "config/Makefile"] {os != "win32"}
+  ["mv" "config/Makefile.mingw" "config/Makefile"] {os = "win32"}
+  [make "all"]
+]
+dev-repo: "git+https://github.com/xavierleroy/camlidl.git"
+extra-files: [
+  ["camlidl.install" "md5=cf56e14faed046880b7c9d0f4cd737f1"]
+  ["META" "md5=2e2f78ac8c3e9f5581a28b2cec6d755b"]
+]
```

Comment:
Do I understand correctly that at every release, `META` will change (to bump the version number), and therefore those checksums must be updated manually?

Is there a way NOT to store these checksums in the .opam file?  After all, we don't checksum individual source files, so why checksum these two?

Reply:
Looking at opam's manual for [`extra-files`](https://opam.ocaml.org/doc/Manual.html#opamfield-extra-files), it seems that the checksums are required for this field. I'll ask around what can be done instead.

Reply:
Ah, I copied the files from opam--that's why it needed the `extra-files`, as they're not distributed with the source tarball; but if we include them in this repo there's no need for this field.